### PR TITLE
improve docstring for TopLevelItemDescriptor.getDescription

### PR DESCRIPTION
**Pull request by jsoref**
grammar: recommended
grammar: plain text
clarify: consistent -- the old text did not really make sense
grammar: article: a
clarify: otherwise

### Desired reviewers

@daniel-beck

### CODE REVIEW

File: core/src/main/java/hudson/model/TopLevelItemDescriptor.java [hunk 1]
```diff
@@ -135,10 +135,10 @@ public String getDisplayName() {
     }
 
     /**
-     * A description of this kind of item type. This description can contain HTML code but it is recommend to use text plain
-     * in order to avoid how it should be represented.
+     * A description of this kind of item type. This description can contain HTML code but it is recommended that
+     * you use plain text in order to be consistent with the rest of Jenkins.
      *
-     * This method should be called in a thread where Stapler is associated, but it will return an empty string.
+     * This method should be called in a thread where a Stapler is associated, otherwise it will return an empty string.
```
Comment (daniel-beck) [inline core/src/main/java/hudson/model/TopLevelItemDescriptor.java]: Stapler is the Jenkins web framework, so `a Stapler` doesn't really work. It's not the office tool.

Perhaps *from a thread handling an HTTP request*? Although I'm not 100% certain these are equivalent.

Comment (jsoref) [reply core/src/main/java/hudson/model/TopLevelItemDescriptor.java]: How about _from a thread where Stapler is handling an HTTP request_? That _sounds_ like what it's trying to say. Admittedly, we could be wrong in understanding how it behaves at edges, but...

Comment (daniel-beck) [reply core/src/main/java/hudson/model/TopLevelItemDescriptor.java]: seems ok.

**Review by oleg-nenashev — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Please avoid force pushes, it complicates the review.
We can always squash-merge PRs from the GitHub interface if needed

**Comment by oleg-nenashev** [comment]
Thanks @jsoref !

**Pull request merged**